pyproject.toml

```diff
@@ -35,6 +35,9 @@ anthropic = "^0.60.0"
 transformers = "^4.51.3"
 torch = "^2.7.0"
 scikit-learn = "^1.7.1"
+easyocr = "^1.7.2"
+paddleocr = "^3.2.0"
+paddlepaddle = "^3.2.0"
 
```

Comment:
Can we consider adding `paddlepaddle-gpu` add a check for gpu and use that if present?
Default: `cpu`

Reply:
* Switching to GPU doesn't seem to be straightforward. We have to install the version of paddlepaddle-gpu that supports the specific version of CUDA in the system (according to [this](https://www.paddlepaddle.org.cn/en/install/quick?docurl=/documentation/docs/en/develop/install/pip/linux-pip_en.html)). I don't see any easy way to add the dependency.
 * Removed
 * I think it should be a dependency of some of the existing libraries. But added it just in case.

Reply:
Ah ok.